docs/creating-pdf-documents.md

```diff
@@ -93,9 +94,31 @@ Ensure that you download a full distribution with appropriate libraries installe
 
 ## Rendering PDFs with Typst
 
-MyST also provides an option to build PDFs with [Typst](https://typst.app). Typst is a markup-based typesetting language. Compared to $\LaTeX$, syntax is streamlined and consistent, and compile time is significantly faster. To render Typst PDFs locally, you must install the [Typst CLI](https://github.com/typst/typst).
+[Typst](https://typst.app) is a markup-based typesetting language. It is **significantly faster and simpler than using $\LaTeX$** with results of equal or better quality.
 
-To add Typst to your export targets, add `format: typst` and select a Typst template. These templates use the same [MyST templating library](xref:jtex) as $\LaTeX$ templates to support document [frontmatter](./frontmatter.md).
+(typst:install)=
+### How to install Typst
+
+Follow [the Typst installation instructions](https://github.com/typst/typst?tab=readme-ov-file#installation) for several options to install Typst.
+We **strongly recommend using the latest releases of Typst**. If you get a confusing Typst error, a good first step is to upgrade your version of Typst.
+
+We recommend installing Typst with Mamba, like so:
```

Comment:
Similar to npm, the conda versions are not maintained by the core team (as far as I can tell). So these will lag behind and/or could be abandoned. My recommendation would be to follow the typst docs directly.

Reply:
ok will remove that

Reply:
I'm in favour of keeping mamba instructions. Many packages on conda-forge are community maintained, and it makes thinks a lot easier if users are pulling other things from conda. I think we focus on what works well right now. Unless we find that the version on forge is already too old for our required feature set.

Reply:
Personally I agree with your point about mamba - I included it originally because it's how we recommend installing myst for Python users, and the conda forge version of typst seemed to be updated.

That said I don't have a lot of mental bandwidth for refining this PR so I'd prefer if we just got this to a stage that was safe to merge and then did it. If folks want to debate whether mamba should make it into the docs then I'm happy for somebody to take over the PR.